### DISCUSSION TITLE
Is it possible to make Dropdowns bigger

### DISCUSSION THREAD

**Discussion by vcdwelt** [cyanfish/naps2#385] [General]
Hi,
i'm using this great App for a long time. It's really great!

But one small thing is really annoying sometimes :-) Every time i use the Small dropdowns beside the big Icons, it is really hard to select a submenu point there.
Is it possible in any way to make this small dropdowns a little big bigger?
![image](https://github.com/cyanfish/naps2/assets/7173074/1fcf99c0-3040-4581-b7bd-0e2058129d3e)

**Comment by cyanfish** [comment]
It's not currently possible, but in your Settings you can set "Scan button default action" to "Always prompt" if you want the bigger part of the button to open the dropdown.

**Reply by vcdwelt** [reply]
Thanks a lot. It seems i missed this option until now :) 
It helps me a lot.

Thank You!
